ydb/core/viewer/viewer_cluster.h

```diff
@@ -342,8 +342,12 @@ class TJsonCluster : public TViewerPipeClient {
     }
 
     void InitSystemWhiteboardRequest(NKikimrWhiteboard::TEvSystemStateRequest* request) {
-        //request->AddFieldsRequired(-1);
-        Y_UNUSED(request);
+        for (auto field : {1, 2, 4, 5, 6, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 30, 31, 36, 37}) { // node_whiteboard.proto default fields
```

Comment:
maybe you want to use their enum names?

Reply:
> maybe you want to use their enum names?

maybe, maybe even form them based on protobuf reflection ... that's a good idea, by the way, thank you!